radicle-cli/src/commands/push.rs

```diff
@@ -0,0 +1,134 @@
+use std::ffi::OsString;
+use std::path::Path;
+
+use radicle::git;
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+use anyhow::anyhow;
+
+pub const HELP: Help = Help {
+    name: "push",
+    description: env!("CARGO_PKG_DESCRIPTION"),
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad push [--all] [--[no-]sync] [<option>...]
+
+    By default, only the current branch is synced.
+
+Options
+
+    --all               Push all branches (default: false)
+    --sync              Sync after pushing to the "rad" remote (default: false)
+    --no-sync           Do not sync after pushing to the "rad" remote
+    --help              Print help
+
+Git options
+
+    -f, --force           Force push
+    -u, --set-upstream    Set upstream tracking branch
+
+"#,
+};
+
+#[derive(Default, Debug)]
+pub struct Options {
+    pub verbose: bool,
+    pub force: bool,
+    pub all: bool,
+    pub set_upstream: bool,
+    pub sync: bool,
+}
+
+impl Args for Options {
+    fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
+        use lexopt::prelude::*;
+
+        let mut parser = lexopt::Parser::from_args(args);
+        let mut verbose = false;
+        let mut force = false;
+        let mut all = false;
+        let mut sync = false;
+        let mut set_upstream = false;
```

Comment:
small suggestion, alphabetical order?

Reply:
I don't think it's worth it for a list of 6 items?

Reply:
Having clear ordering makes it easier to add items.  Newcomers have to figure out if there's a hidden order or not when adding fields.  Its stylistic and not a big deal, but I find it reduces a small amount of friction.